packages/manager/apps/public-cloud/src/index.routes.js

```diff
@@ -22,8 +22,10 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
   $stateProvider.state('app.redirect', {
     url: '?onboarding',
     resolve: {
-      redirect: /* @ngInject */ ($state) =>
-        $state.go('pci.projects.onboarding'),
+      redirect: /* @ngInject */ ($state, coreConfig) =>
+        coreConfig.getRegion() === 'US'
+          ? $state.go('pci.projects.onboardingUs')
+          : $state.go('pci.projects.onboarding'),
```

Comment:
I think you use the method redirectTo: (transition) => {

Reply:
Is there a reason to refactor the `redirect` function of the `plublic-cloud` router ?